src/optim_problems/opt_cep.jl

```diff
@@ -1,45 +1,66 @@
-# optimization problems
+# CEP
 """
     setup_cep_opt_sets(ts_data::ClustData,opt_data::CEPData)
 fetching sets from the time series (ts_data) and capacity expansion model data (opt_data) and returning Dictionary with Sets as Symbols
 """
 function setup_opt_cep_set(ts_data::ClustData,
                             opt_data::OptDataCEP,
-                            opt_config::Dict{String,Any};
-                            k_ids::Array{Int64}=Array{Int64,1}())
+                            opt_config::Dict{String,Any})
+  #`costs::OptVariable`: costs[tech,node,year,account,impact] - annulized costs [USD in USD/MW_el, CO2 in kg-CO₂-eq./MW_el]`
+  costs = opt_data.costs
+  #`techs::OptVariable`: techs[tech] - OptDataCEPTech
+  techs = opt_data.techs
+  #`nodes::OptVariable`: nodes[tech,node] - OptDataCEPNode
+  nodes = opt_data.nodes
+  #`lines::OptVarible`: lines[tech,line] - OptDataCEPLine
+  lines = opt_data.lines
+
   set=Dict{String,Array}()
-  set["nodes"]=unique(opt_data.nodes[:nodes])
+  set["nodes"]=axes(nodes,"node")
```

Comment:
OptVariables are fancy now:
axes(OptVariable,String) gives the corresponding set